tools/travis-scripts/run-script.sh

```diff
@@ -291,6 +301,12 @@ function run_pull_request()
         build_linux
     fi
 
+    # linux_clang_tidy_test
+    if [ $BUILD_TARGET == 'linux_clang_tidy' ]; then
+        genernate_binding_codes
```

Comment:
I think that genernate_binding_codes should be removed

Reply:
Yep, it is not needed.